src/messagingapplication.cpp

```diff
@@ -289,6 +295,13 @@ void MessagingApplication::parseArgument(const QString &arg)
             QMetaObject::invokeMethod(mainView, "startChat", Q_ARG(QVariant, properties));
         }
     }
+    if (scheme == "sms") {
```

Comment:
can't it be grouped with the `if (scheme == "message")`, e.g: `if (scheme == "message" || scheme == "sms")` instead of duplicating the logic ?

Reply:
awesome will do :)